parquet-column/src/main/java/org/apache/parquet/column/ParquetProperties.java

```diff
@@ -199,6 +215,8 @@ public static Builder copy(ParquetProperties toCopy) {
     private int pageSize = DEFAULT_PAGE_SIZE;
     private int dictPageSize = DEFAULT_DICTIONARY_PAGE_SIZE;
     private boolean enableDict = DEFAULT_IS_DICTIONARY_ENABLED;
+    private boolean enableBloomFilter = DEFAULT_BLOOM_FILTER_ENABLED;
+    private HashMap<String, Long> bloomFilterColumnExpectedNDV = new HashMap<>();
```

Comment:
We usually don't use `HashMap` as a reference type but `Map` so the implementation is hidden. Is it intentional that you `HashMap` as reference type and even return this reference in a public method?